src/service/resources-impl.js

```diff
@@ -254,6 +258,22 @@ export class Resources {
       this.schedulePass();
       this.monitorInput_();
     });
+
+    // Safari 10 and under incorrectly estimates font spacing for `@font-face`
```

Comment:
We could. Overall, this is not a horrible idea and expense is likely small - we only remeasure things here. Do you think it's worth to guard this by Safari only? If so, I'll add.